examples/allocation-endpoint/README.md

```diff
@@ -0,0 +1,104 @@
+# Allocation Endpoint
+
+This is a sample solution to enable an Allocation Endpoint proxy in front of GKE Agones clusters.
+
+In this folder you will find:
+
+1. Terraform modules that creates resources in GCP
+2. The Allocation Endpoint Proxy code that gets deployed to Cloud Run
+3. An ESP container that gets deployed to agones-allocator
+4. A client sample code to send allocation requests to the proxy
+5. Dockerfile that builds the proxy and scripts to push the image to a docker repository
+6. Documentation on how to use the solution.
+
+Here is the architecture of GCP resources created:
+
+![architecture](https://github.com/googleforgames/agones/blob/main/examples/allocation-endpoint/architecture.png?raw=true)
```

Comment:
Does this work as a relative link? The link is broken while previewing the page (but will probably be fine once the commit goes in).